extensions/applicationinsights-clickanalytics-js/src/handlers/AutoCaptureHandler.ts

```diff
@@ -58,7 +58,13 @@ export class AutoCaptureHandler implements IAutoCaptureHandler {
         
             // Process click event
             function _processClick(clickEvent: any) {
-                var clickCaptureElements = { A: true, BUTTON: true, AREA: true, INPUT: true };
+                var clickCaptureElements: Record<string, boolean> = { A: true, BUTTON: true, AREA: true, INPUT: true };
```

Comment:
Let move this initialization to the init in an onConfigChange() so we only process this once. rather than creating a new object every time and then "updating" it.

Either that or we change the default set (the config for trackElementTypes) to just be a string[] and just look it up here.